.github/workflows/kuksa_databroker-cli_build.yml

```diff
@@ -72,7 +72,7 @@ jobs:
 
   build:
     name: Build databroker-cli
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Build failed suddenly due to the use of ubuntu-latest with the following message: 

```
Run cargo install cross cargo-license cargo-cyclonedx
    Updating crates.io index
     Ignored package `cross v0.2.5` is already installed, use --force to override
    Updating crates.io index
     Ignored package `cargo-license v0.6.1` is already installed, use --force to override
    Updating crates.io index
     Ignored package `cargo-cyclonedx v0.5.5` is already installed, use --force to override
     Summary Successfully installed cross, cargo-license, cargo-cyclonedx!
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.12/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
Error: Process completed with exit code 1.
```

The following comment probably describes the problem best: https://git hub.com/PyO3/maturin-action/issues/291#issue-2564997789

We can either add --break-system-packages or downgrade the image back to 22.04 - I decided to pin the version to ubuntu-22.04 for the build step as an intermediate step to resolve the issue.